app/cafe.py

```diff
@@ -0,0 +1,24 @@
+import datetime
+from .errors import (NotVaccinatedError,
+                     NotWearingMaskError,
+                     OutdatedVaccineError)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        today_date = datetime.date.today()
+
+        if "vaccine" not in visitor or not visitor.get("vaccine"):
```

Comment:
seems like a bit overcomplicated statement, isn't it ?

Reply:
what will return `visitor.get("vaccine")` if vaccine not in visitor?

Reply:
Ah, this is overcomplicated statement. 
Fixed